parse.go

```diff
@@ -37,8 +37,9 @@ type Import struct {
 }
 
 type Option struct {
-	Name  string `json:"name,omitempty"`
-	Value string `json:"value,omitempty"`
+	Name          string   `json:"name,omitempty"`
+	Value         string   `json:"value,omitempty"`
+	NestedOptions []Option `json:"values,omitempty"`
```

Comment:
I think the name `NestedOptions` could be more descriptive -- initially this confused me a bit. Is it truly "nested" or are they just more complex options?  

Also, is it possible to have an `Option` with a `Name`, `Value`, _and_ `NestedOptions` set?

Reply:
The official name of this form is "Aggregated". So maybe AggregatedOptions? emicklei calls them AggregatedConstants